### PR TITLE
Request Path service

### PR DESCRIPTION
**Pull request by gus-robotics88**
See https://github.com/robotics-88/path-manager/pull/18

### CODE REVIEW

File: path_planning/src/path_planning.cpp [hunk 1]
```diff
@@ -85,8 +89,12 @@ class planner {
 			std::cout << "Goal point set to: " << x << " " << y << " " << z << std::endl;
 
 			if(set_start)
-				plan();
+				return plan();
+		}
+		else {
+			std::cout << "Path planner goal same as previous goal, not planning" << std::endl;
```
Comment (rien88) [inline path_planning/src/path_planning.cpp]: Don't we send the same goal more than once with the service, if it fails?

File: path_planning/src/path_planning.cpp [hunk 2]
```diff
@@ -337,13 +345,19 @@ class planner {
 				// vel_pub->publish(minjerk_velocity);
 				// acc_pub->publish(minjerk_accel);
 
+				// Set last path to kino nav path
+				last_path_ = kino_nav_path;
+
     			path_size_int64.data = kino_path.size()+1;
 				path_size_pub->publish(path_size_int64);
```
Comment (rien88) [inline path_planning/src/path_planning.cpp:348]: creative indenting?

File: path_planning/src/path_planning.cpp [hunk 3]
```diff
@@ -474,9 +501,13 @@ int main(int argc, char **argv)
 	auto odom_sub = node->create_subscription<nav_msgs::msg::Odometry>("/mavros/odometry/out", 1, odomCb);
 	auto pointcloud_sub = node->create_subscription<sensor_msgs::msg::PointCloud2>(cloud_topic, 1, cloudCallback);
 	auto pose_sub = node->create_subscription<geometry_msgs::msg::PoseStamped>(pose_topic_, 100, poseCb);
-	auto goal_sub = node->create_subscription<geometry_msgs::msg::PoseStamped>("/goal", 10000, goalCb);
+	// auto goal_sub = node->create_subscription<geometry_msgs::msg::PoseStamped>("/goal", 10000, goalCb);
```
Comment (rien88) [inline path_planning/src/path_planning.cpp]: Delete? same re the goalCb function

File: path_planning/src/path_planning.cpp (hunk 2 above)
Comment (gus-robotics88) [reply path_planning/src/path_planning.cpp:348]: There's creative indenting all over this file, I don't think I added any new creative indenting though.

File: path_planning/src/path_planning.cpp (hunk 1 above)
Comment (gus-robotics88) [reply path_planning/src/path_planning.cpp]: True, got rid of this conditional.

File: path_planning/src/path_planning.cpp (hunk 2 above)
Comment (rien88) [reply path_planning/src/path_planning.cpp:348]: Haha fair enough :laughing:

**Review by rien88 — APPROVED** [review]

**Pull request merged**